### DISCUSSION TITLE
How to promoted label only for children

### DISCUSSION THREAD

**Discussion by shirobachi** [TriliumNext/Trilium#3286] [Q&A] [answered]
Hello, 

I would like to make a promoted label, but only for children of the note, not the note itself. Is there a way to do it?
I tried to name it with `child:` prefix, but it did not work ;/

Thanks in advance!

**Comment by zadam** [comment, marked as answer]
`child:` should do what you want, but this is a "copy inheritance" - these attributes are copied when a new note is created under the given parent.

You might also consider pattern to create a [template](https://github.com/zadam/trilium/wiki/Template) with the promoted attributes and then create `~child:template=[your template]`.

**Reply by shirobachi** [reply]
Thanks for this! I tried with child, and it does not work well. Could you [have a look](https://i.imgur.com/wfo1ntX.png) if I try it correctly?


Thanks for your product, I love it! <3

**Reply by zadam** [reply]
The mistake is in the wrong order - it should be `child:` first, e.g. `#child:label:myLabel=promoted,single,text`

**Reply by shirobachi** [reply]
Yeah, that works! Thanks a lot!!

**Reply by shirobachi** [reply]
Hey @zadam,
I would like to ask for tip. I would like to have label definition only for direct children and to adding new label definition affect for notes made in the past? Is there a way to do this smart?

**Reply by zadam** [reply]
@Shirobachi no. Normal inheritance works for old children, but does not have a way to limit depth. "copy" inheritance limits depth, but does not work automatically on old notes. You could use "bulk actions" to fill in the attributes to older notes though:

<img width="951" alt="image" src="https://user-images.githubusercontent.com/617641/202809960-d932be16-dd77-4ed3-befa-9c35213ea422.png">

**Reply by shirobachi** [reply]
Thanks so much, I did not know about bulk action, that will work for me I think <3
